package.json

```diff
@@ -58,8 +58,7 @@
     "eslint-config-okonet": "^5.0.1",
     "eslint-plugin-node": "^5.1.1",
     "husky": "^0.14.3",
-    "jest": "^21.1.0",
-    "jest-cli": "^21.1.0",
```

Comment:
@okonet Was facing some issue and that was why we had the `jest-cli` in dev-dependencies. @okonet, could you check if the issue still persists? Could have been fixed with the changes since v21.

Reply:
The `jest` package itself is just `jest-cli`: https://github.com/facebook/jest/tree/master/packages/jest

If you need both it's a bug I'd love to fix 🙂

Reply:
Yes, @luftywiranda13 pointed that out in https://github.com/okonet/lint-staged/pull/141#discussion_r136150058. My guess would be that it is an npm issue.